FeedingHopeAndHeroes/FeedingHopeAndHeroes.html

```diff
@@ -0,0 +1,201 @@
+          </p>
+          <p class="donation-text">
+            If you would like to donate via our JustGiving page click here...
+          </p>
+          <button class="donate-button">
+            Donate
+          </button>
+        </div>
+      </div>
+      <img
+        src="Images/0ff7807a-4117-4ae1-98ce-824e1539d358(cropped).png"
+        class="right-half top-img-area"
+      />
+    </div>
+    <div class="feed-our-frontline-section">
+      <img
+        src="Images/8e95d135-6f51-40af-90de-3e5866b56959(cropped).jpg"
+        class="left-half frontline-img"
+      />
+      <div class="right-half frontline-text-area">
+        <div class="frontline-container">
+          <h1 class="title">
+            <img src="feedFrontlineTitle.PNG" style="height: 200px;" />
+          </h1>
+          <p class="frontline-text">
+            Providing prepared meals and fresh fruit/veg boxes to NHS staff
+            during the COVID-19 crisis. Our aim is to service NHS hospitals
+            across the UK this crisis.
+          </p>
+        </div>
+      </div>
+    </div>
+
+    <div class="donation-form-section">
+      <h1 class="donation-form-header">
+        If you are a food supplier or restaurant who would like to donate
+      </h1>
+      <div class="form-container">
+        <div class="form-left">
+          <form>
+            <div class="form-col1">
+              <label class="donation-label" for="fname">Full Name</label><br />
+              <input type="text" class="fname" name="fname" value="" /><br />
+              <label class="donation-label" for="company">Company</label><br />
+              <input
+                type="text"
+                class="company"
+                name="company"
+                value=""
+              /><br />
+              <label class="donation-label" for="email">Email Address</label
+              ><br />
+              <input type="text" class="email" name="email" value="" /><br />
+              <label class="donation-label" for="contact">Contact Number</label
+              ><br />
+              <input
+                type="text"
+                class="contact"
+                name="contact"
+                value=""
+              /><br />
+            </div>
+            <div class="form-col2">
+              <label class="donation-label" for="address">Address</label><br />
+              <textarea
+                type="text"
+                class="address"
+                name="address"
+                value=""
+              /></textarea><br />
+              <label class="donation-label" for="message">Message</label><br />
+              <textarea
+                type="text"
+                class="message"
+                name="message"
+                value=""
+              /></textarea><br />
+              <button type="submit" class="donation-button form-submit-button">
+                Thank you
+              </button>

```

Comment:
Maybe add a small piece of text explaining what the submitter should expect next. Phone call? Email? Site visit? and when? In the next 2 days or so?

Reply:
This is a placeholder made while waiting for final design